mods/pokebattler.php

```diff
@@ -108,18 +102,17 @@
     // Raid tier array
     debug_log('Processing the following raid levels:');
     $raidlevels = array();
-    for($i = $start; $i <= $end; $i++) {
-        $rl = $i + 1;
-        if($rl == 6) $rl = 'MEGA';
-        $raidlevels[] = 'RAID_LEVEL_' . $rl;
+    foreach($get_levels as $level) {
+        if($level == 6) $level = 'MEGA';
```

Comment:
if($level == 6) {
    $level = 'MEGA';
}

Reply:
If you’re asking WHY?, let me just say „for a good reason...“

Reply:
Yeah, I just shuffled the lines around, happy to fix it to be more readable. :-)